codes/models/alt_growth_rates/FRA_adm1.do

```diff
@@ -10,6 +10,8 @@ cap set scheme covid19_fig3 // optional scheme for graphs
  
 // set up time variables
 gen t = date(date, "YMD")
+keep if t <= date("20200325","YMD")
```

Comment:
We're removing hard coded cutoff dates. This should be read from `codes/data/cutoff_dates.csv` (see #29). I will merge that PR as soon as we've got a start-to-finish working version of the current repo (almost there) so that we can tag it as our "initial submission" data.

Reply:
merging this now but flagging that we want to adjust this soon